test/containers/MasonryExample.js

```diff
@@ -57,7 +57,7 @@ export default class MasonryExample extends React.Component {
       window.RESIZE_MEASUREMENT_DONE = false;
       setTimeout(() => {
         const checkIfMeasuring = () => {
-          if (!this.gridRef.state.hasPendingMeasurements) {
+          if (!this.gridRef.hasPendingMeasurements) {
```

Comment:
Can this use the public API rather than reading internal measurements now?

Reply:
done